rele/config.py

```diff
@@ -102,5 +102,10 @@ def load_subscriptions_from_paths(sub_module_paths, sub_prefix=None, filter_by=N
             if filter_by and not subscription.filter_by:
                 subscription.set_filters(filter_by)
 
-            subscriptions.append(subscription)
-    return subscriptions
+            if subscription.name in subscriptions:
+                raise RuntimeError(
```

Comment:
Do we want to raise or skip and log? If we skip, the behaviour will be [similar](https://github.com/mercadona/rele/blob/master/rele/client.py#L72) to what happens when there is no topic for the subscription.

Reply:
I'd raise because it's not something that should be done. A duplicate sub will only bring pain(either splitting consumption or if we skip not knowing why a given sub is not executed)